test_plans/hadoop-processing-lxd.yaml

```diff
@@ -0,0 +1,7 @@
+bundle: bundle:~/juju-qa/hadoop-processing
+benchmark:
+    resourcemanager:
+        terasort
+bundle_name: hadoop-processing-lxd
+bundle_file: hadoop-processing-lxd.yaml
```

Comment:
Please change this to `bundle.yaml`. This is needed when the bundle have multiple bundle files such as `bundle-dev.yaml` and `bundle.yaml`, need this to be explicit which bundle file to use . To be exact, please change it to:
`bundle_file: bundle.yaml`